test/sharness/t0030-mount.sh

```diff
@@ -20,14 +20,10 @@ test_launch_ipfs_daemon
 
 # run this mount failure before mounting properly.
 
-test_expect_failure "'ipfs mount' fails when no mount dir (issue #341)" '
-	test_must_fail ipfs mount -f=not_ipfs -n=not_ipns >actual
-'
-
-test_expect_failure "'ipfs mount' looks good when it fails (issue #341)" '
-	! grep "IPFS mounted at: $(pwd)/ipfs" actual >/dev/null &&
-	! grep "IPNS mounted at: $(pwd)/ipns" actual >/dev/null ||
-	test_fsh cat actual
```

Comment:
> The set of all possible outputs that don't contain "IPFS mounted at:" and "IPNS mounted at:" is too broad. What if someone changes our output message to "IPFS mounted at" (without the ":") and then someone introduces a bug that causes ipfs mount to print our output message? The test will pass. The test will pass under almost any circumstances. And it haven't helped us the last time.

It is critical that we test our output formats as exactly as we can, so that if we accidentally change the output format we _notice_. People will build scripts on top of our outputs and they will only be able to do so as long as we do not change the outputs unless absolutely necessary. This should be a very explicit decision, and test cases help us make sure we do this carefully.